### PR TITLE
Create faq.md

### PR DESCRIPTION
**Pull request by tasodorff**
Added a FAQ file for working with the ontology.

### CODE REVIEW

File: ontology/docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Frequently Asked Questions
+
+This document is intended to provide answers to frequently asked questions regarding the application of the Digital Buildings ontology to existing and novel (i.e. previously unmodeled) devices.
+
+[TOC]
+
+## What is a building model?
+**A building model is a file that maps concrete real-world devices to the ontology.** 
+
+The model takes the form of a building configuration file (see [here](building_config.md)). It contains all the relevant information for the building: what devices exist, what types they apply, which devices connect to which other devices, what devices serve which zone, etc.
```
Comment (charbull) [inline ontology/docs/faq.md:10]: might need ./building_config.md

File: ontology/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+## Why have a building model?
+**It makes building data useful.**
```
Comment (charbull) [inline ontology/docs/faq.md:13]: makes building data unified across buildings?

File: ontology/docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## What data should be made available at the gateway (or controller) level?
+**Everything, insofar as is practical.**
+
+It does not have to be translated into the model, but for extensibility purposes, all practical data should be exposed. The reason for this is that it is sometimes difficult (or impossible) to programmatically expose data once the system is configured. Certain BAS platforms are easy to expose after-the-fact (e.g. ALC) but some gateways require on-site work to expose, so better to be exposed and not used than not exposed and needed later on. Updating the translation layer is much easier in most cases than exposing more data in the gateway.  
+
+## What device data should be modeled?
+**Model what you need for the applications and analytics that are reasonably anticipated.**
```
Comment (charbull) [inline ontology/docs/faq.md:48]: we dont have applications/analytics provided in most cases to the MSi, can we provide a default mode more clearly.
you mention what is practical. Can we list namespaces or functionalties?
For example, should someone model fire systems? or badge reader?

File: ontology/docs/faq.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+### Control States
+These are values that represent how the device is responding to measured deviations from setpoints or other changes in environmental/temporal conditions. Such fields as `supply_fan_run_command` and `chilled_water_valve_percentage_command` are examples of control states that the device may adjust directly or indirectly based on measured conditions (e.g. the time of day for scheduled operation of the fan, or setpoint deviation for position of the valve). Most control states are normally modeled.
+
+### Interpreted States
+These are states which the device interprets from underlying data (whether it is exposed or not). The best example of this is an alarm: for example, a fan mismatch alarm compares the status of the fan to the command and returns an alarm if they are no equal. Interpreted states are not normally modeled.
+
+### Configuration Information
+These are internal device points which are used to configure the way in which the device attempts to maintain control values, or how its components control internally. Stage up/down timers, PID gains, etc.; configuration information is rarely modeled. 
+
+## How do you extend the ontology?
```
Comment (charbull) [inline ontology/docs/faq.md:67]: can we mentioned that the ontology type validator is triggered upon a pull request.
Keep iterating until the Ontology Type validator passes which means that the content is valid?

File: ontology/docs/faq.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+## What is the process for accepting extensions to the ontology?
+**If the extension passes the type validator and passes review by the team, it will be accepted.** 
+
+The type validator checks that additions or changes to the ontology do not break existing rules or compatibility (for example, the type validator will raise errors if a new field is added which is composed of undefined subfields). A review of the proposed extensions will be performed by the Digital Buildings team, which include building systems subject matter experts and software developers. The review covers both the subject matter (e.g. is a subfield too broadly defined for the intended application) and its holistic effect on the ontology (e.g. does it set precedent that makes further extension difficult). These reviews may require some deliberation and can result in proposed changes to the pull request, so it is better to submit pull requests in phases. See the next question for what is advised on ordering of ontology updates.
```
Comment (charbull) [inline ontology/docs/faq.md:95]: cool I see it here. Should we add a section here on how this works? from here: http://go/digitalbuildings-sdk

**Review by charbull — CHANGES_REQUESTED** [review]
minor changes and mainly questions

File: ontology/docs/faq.md (hunk 1 above)
Comment (tasodorff) [reply ontology/docs/faq.md:10]: i dont think it does... we dont have those in the links in other ontology docs in the same folder.

File: ontology/docs/faq.md (hunk 2 above)
Comment (tasodorff) [reply ontology/docs/faq.md:13]: added.

File: ontology/docs/faq.md (hunk 3 above)
Comment (tasodorff) [reply ontology/docs/faq.md:48]: we could, if we knew what they were. we are integrating fire systems today, so i assume there is a reason... i just dont know it, and therefore cant advise on what is needed or not. who in google would know why? what is their expectation for what is being monitored or analyzed?

File: ontology/docs/faq.md (hunk 4 above)
Comment (tasodorff) [reply ontology/docs/faq.md:67]: added

File: ontology/docs/faq.md (hunk 5 above)
Comment (tasodorff) [reply ontology/docs/faq.md:95]: link doesnt work for me, not on my google machine... assume this is an internal doc?

**Pull request merged**